crates/ra_syntax/src/grammar.ron

```diff
@@ -411,6 +411,7 @@ Grammar(
         "PrefixExpr": (),
         "RangeExpr": (),
         "BinExpr": (),
+        "String": (),
```

Comment:
Is there any way we can avoid the name `String` here (bikeshed: StringLit, StringExpr etc.)?

Reply:
Not sure... The token is called `STRING` and as far as I know the generator requires that you use the token name in CamelCase. Maybe @matklad knows?